pkg/telemetry/prometheus/packets.go

```diff
@@ -17,15 +17,16 @@ const (
 )
 
 var (
-	bytesIn           atomic.Uint64
-	bytesOut          atomic.Uint64
-	packetsIn         atomic.Uint64
-	packetsOut        atomic.Uint64
-	nackTotal         atomic.Uint64
-	retransmitBytes   atomic.Uint64
-	retransmitPackets atomic.Uint64
-	participantJoin   atomic.Uint64
-	participantRTC    atomic.Uint64
+	bytesIn            atomic.Uint64
+	bytesOut           atomic.Uint64
+	packetsIn          atomic.Uint64
+	packetsOut         atomic.Uint64
+	nackTotal          atomic.Uint64
+	retransmitBytes    atomic.Uint64
+	retransmitPackets  atomic.Uint64
+	participantJoin    atomic.Uint64
```

Comment:
Made a note in `protocol` PR about renaming `participant_rtc` -> `participant_rtc_connected`. Along the same lines, wondering if we should rename `participant_join` -> `participant_signal_connected` to make things more explicit/clear.